assets/scss/core/_base.scss

```diff
@@ -68,10 +66,88 @@ textarea {
   border-radius: $dp-border-radius;
   font-size: $dp-font-size-form;
   border: 1px solid $dp-color-sylver;
+
+  &::placeholder {
+    color: $dp-color-default;
+    font-style: italic;
+  }
+
+  :focus {
+    border: 1px solid $dp-color-yellow;
+  }
+}
+/* validate form*/
+.field-group {
+  position: relative;
+  display: inline-block;
+  text-align: left;
+  width: 100%;
+  margin-bottom: 30px;
 }
 
-textarea:focus {
-  border: 1px solid $dp-color-yellow;
+.error-messages {
+  display: none;
+}
+
+.error {
+  input,
+  select,
+  textarea {
+    border: 1px solid $dp-color-red;
+  }
+
+  .error-messages {
+    display: inline-block;
+
+    p {
+      margin: 0;
+      color: $dp-color-red;
+      font-size: $dp-font-size-small;
```

Comment:
What do you think about adding these properties to `.error-messages` and not only to `error-messages p`?

I do not know if `p` will be always the right way of representing this information.